app/controllers/agentAppointment/AddAgentController.scala

```diff
@@ -16,31 +16,41 @@
 
 package controllers.agentAppointment
 
+import javax.inject.{Inject, Named}
 import actions.AuthenticatedAction
 import actions.agentrelationship.WithAppointAgentSessionRefiner
+import actions.agentrelationship.request.AppointAgentSessionRequest
+import actions.requests.BasicAuthenticatedRequest
 import binders.pagination.PaginationParameters
 import binders.propertylinks.{ExternalPropertyLinkManagementSortField, ExternalPropertyLinkManagementSortOrder, GetPropertyLinksParameters}
 import config.ApplicationConfig
 import connectors.AgentsConnector
-import controllers.PropertyLinkingController
-import controllers.agentAppointment.AppointNewAgentForms._
-import javax.inject.{Inject, Named}
-import models.propertyrepresentation
+import controllers.agent.routes
+import controllers.{PaginationParams, PropertyLinkingController}
+import form.{EnumMapping, Mappings}
+import models.{RepresentationApproved, RepresentationPending, propertyrepresentation}
+import models.propertyrepresentation.{AddAgentOptions, AppointAgentRequest, AppointNewAgentSession, AppointmentScope, ChooseFromList, ManagingProperty, SearchedAgent, SelectedAgent, Yes}
 import models.propertyrepresentation.AppointAgentRequest.submitAppointAgentRequest
-import models.propertyrepresentation._
-import models.searchApi.AgentPropertiesFilter.Both
-import models.searchApi.AgentPropertiesParameters
-import play.api.data.FormError
+import models.searchApi.AgentPropertiesFilter.{Both, No}
+import models.searchApi.AgentPropertiesSortField.Address
+import play.api.data.{Form, FormError, Forms, Mapping}
+import play.api.data.Forms.{boolean, longNumber, mapping, nonEmptyText, optional, single, text}
 import play.api.i18n.MessagesApi
 import play.api.mvc.{Action, AnyContent, MessagesControllerComponents}
 import repositories.SessionRepo
 import services.AgentRelationshipService
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.play.HeaderCarrierConverter
+import controllers.agent.routes
+import models.searchApi.AgentPropertiesParameters
+import views.html.propertyrepresentation.appoint._
+import controllers.agentAppointment.AppointNewAgentForms._
 import uk.gov.hmrc.propertylinking.errorhandler.CustomErrorHandler
 
 import scala.concurrent.{ExecutionContext, Future}
 import scala.util.Try
 
-class AgentRelationshipController @Inject()(
+class AddAgentController @Inject()(
```

Comment:
Looks like a lot of unnecessary {}'s in this file.

Reply:
I have added braces to help with readability. I think they make this clearer in this case. Which ones particularly bother you?

Reply:
You may have noticed that braces often get removed other devs? My impression is that the common standard on this team is to avoid unnecessary {}'s. You added braces where they weren't before. The next dev may remove them. Where does it end :) Not a big deal obviously, and none of them "particularly bother" me.

Reply:
these are unnecessary braces that even intellij highlights to you with a hint to "Remove redundant braces". As per official Scala 
Style Guide https://docs.scala-lang.org/style/control-structures.html#curlybraces 
> case - Always omit braces in case clauses.

![image](https://user-images.githubusercontent.com/1763911/76944331-bbca5900-68f8-11ea-8763-2540f25db8ed.png)